app/delivery/rover_routes.js

```diff
@@ -0,0 +1,20 @@
+import express from "express";
+import getMarsPhotos from "../useCases/fetch_rover_photos.js";
+import Exception from "../utils/exception.js";
+
+
+const roverRouter = express.Router();
+
+roverRouter.post('/rover', async (req, res, next) => {
+    try {
+        const {user_id, user_name, user_api_key} = req.body;
+
+        const photos = await getMarsPhotos(user_api_key)
```

Comment:
Queries directly use user_api_key. This may be unsafe. you can add Authorization headers instead of passing keys through the request body.